static/app/utils/profiling/hooks/useContextMenu.tsx

```diff
@@ -39,66 +39,55 @@ export function useContextMenu({container}: UseContextMenuOptions) {
 
   // We wrap the setOpen function in a useEffect so that we also clear the keyboard
   // tabIndex when a menu is closed. This prevents tabIndex from being persisted between render
-  const wrapSetOpen = useCallback(
```

Comment:
Any reason you removed all the `useCallback` calls?

Reply:
Added them back, I was testing to see if there was a stale closure and forgot to  add it back